frappe/integrations/doctype/oauth_bearer_token/oauth_bearer_token.py

```diff
@@ -28,3 +30,11 @@ class OAuthBearerToken(Document):
 	def validate(self):
 		if not self.expiration_time:
 			self.expiration_time = add_to_date(self.creation, seconds=self.expires_in, as_datetime=True)
+
+	@staticmethod
+	def clear_old_logs(days=30):
```

Comment:
Also enable this by default?

Reply:
Should it be part of `LOG_DOCTYPES`?

> If any logtype table grows too large then clearing it with DELETE query
	is not feasible in reasonable time. This command copies recent data to new
	table and replaces current table with new smaller table.